packages/sdk-cli/src/models/localRelay/relayInfo.test.ts

```diff
@@ -161,19 +154,26 @@ describe('pollRelayInfo', () => {
   });
 
   // First poll request's execution time 2000ms exceeds the total specified 1000ms timeout.
-  it('rejects on timeout', async () => {
+  it('rejects on timeout', (done) => {
     const timeout = 1000;
     readJsonFileSpy = jest.spyOn(util, 'readJsonFile').mockImplementation(
       () =>
         new Promise<false>((resolve) => setTimeout(() => resolve(false), 2000)),
     );
 
-    const poll = pollRelayInfo(timeout, undefined);
+    pollRelayInfo(timeout, undefined)
+      .then(() => done(new Error('Expected pollRelayInfo to timeout')))
+      .catch((error) => {
+        expect(error).toEqual(
+          expect.objectContaining({
+            message: `Timed out after waiting for ${timeout} ms`,
+          }),
+        );
+
+        return done();
```

Comment:
nit: I think you could use `.rejects.` on the result of the promise here to make this a little clearer

Reply:
That was actually the case before this commit. The issue is, I think using promises like this:

```
const promise = fnThatUsesTimeoutsAndPromises();
jest.runAllTimers();
await expect(promise).rejects...
```

is what causes the

```
This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "Error: Timed out after waiting for 240 ms".]
```

It seems like, indeed, the error in the prev CI run was caused by this test, now it is fixed, and the same error is now causes by another test, 'polls in regular intervals'. I will try changing the syntax to the current `then-catch` one, we'll see if the CI succeeds; if it doesn't, I'll change the syntax to `await expect().rejects` and will need to continue looking into what causes the error.

Reply:
Are you sure about this? Because this doesn't seem like a Jest timeout error – this error message is from `async-wait-for` library, and it seems like the promise was rejected without a `try-catch` or `.catch()` clause

Reply:
This error?

```[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "Error: Timed out after waiting for 240 ms".] {
  code: 'ERR_UNHANDLED_REJECTION'
}```?

It comes from Node, not Jest.

Reply:
https://github.com/Fitbit/developer-bridge/pull/176/commits/1b5288515886cabe2144f191fece5e902354df49